src/python/grpcio/grpc/_cython/_cygrpc/credentials.pxd.pxi

```diff
@@ -26,9 +26,9 @@ cdef int _get_metadata(
     grpc_credentials_plugin_metadata_cb cb, void *user_data,
     grpc_metadata creds_md[GRPC_METADATA_CREDENTIALS_PLUGIN_SYNC_MAX],
     size_t *num_creds_md, grpc_status_code *status,
-    const char **error_details) except * with gil
+    const char **error_details) except -1 with gil
```

Comment:
Followed the suggestion from https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#error-return-values to add specific value for exception